sources/OpenMcdf.Extensions/OLEProperties/PropertyFactory.cs

```diff
@@ -412,26 +412,86 @@ public override string ReadScalarValue(System.IO.BinaryReader br)
                 uint size = br.ReadUInt32();
                 data = br.ReadBytes((int)size);

```

Comment:
If it actually is possible for ```size``` to be 0, then is there any value in just returning an empty string in that case, and skipping the GetEncoding/GetString calls?